pbft/support/scripts/build.sh

```diff
@@ -3,11 +3,18 @@ set -x
 
 export GOOS=$1
 export GOARCH=amd64
+export PROJECTPATH="$GOPATH/src/github.com/truechain/truechain-consensus-core/pbft"
+
+cd $PROJECTPATH
```

Comment:
I'm presuming you had another reason for PROJECTPATH? Because in normal case, we'd expect hmake to build this.. eh?

Reply:
Yes. True. As you mentioned I am still using build.sh to build the binaries. I had added this to have build.sh as executable from any level in the project tree. This should eventually be removed and have hmake build it for us.